api/client/logs.go

```diff
@@ -16,7 +16,7 @@ func (cli *DockerCli) CmdLogs(args ...string) error {
 		cmd    = cli.Subcmd("logs", "CONTAINER", "Fetch the logs of a container", true)
 		follow = cmd.Bool([]string{"f", "-follow"}, false, "Follow log output")
 		times  = cmd.Bool([]string{"t", "-timestamps"}, false, "Show timestamps")
-		tail   = cmd.String([]string{"-tail"}, "all", "Number of lines to show from the end of the logs")
+		tail   = cmd.String([]string{"-tail"}, "latest", "Number of lines to show from the end of the logs")
```

Comment:
add `latest` and `all` in the description. I suggest something like:
```
Last number of lines to show. Use `latest` (default) for latest file, or `all` for all logs.
```

@moxiegirl feel free to chime in